swift/IceGrid/simple/README.md

```diff
@@ -0,0 +1,55 @@
+This demo illustrates the basics of using [IceGrid][1], including the use of
+[templates][2] and [object adapter replication][3].
+
+To run the demo, first start the IceGrid service:
+
+```
+icegridnode --Ice.Config=config.grid
+```
+
+Note that for debug Windows builds you will need to use icegridnoded
```

Comment:
This sentence should be removed, I'll create a bug to fix the other README files